lib/seahorn/BvOpSem2Context.hh

```diff
@@ -729,6 +731,14 @@ public:
   /// \brief given a properly encoded pointer Expr \p p , return the raw
   /// expression representing memory address only
   virtual Expr ptrToAddr(Expr p) = 0;
+
+  /// \brief given an Expression \p e , return true if \p e has expected
+  /// encoding of a MemValTy
+  virtual bool isMemVal(Expr e) = 0;
+
+  /// \brief given a properly encoded memory map Expr \p p , return the base
+  /// expression representing raw memory only
+  virtual Expr getRawMem(Expr p) = 0;
```

Comment:
if `p` is `NULL` will we crash or return NULL?

I think some Memory managers crash and others return `NULL` expression('RawMemManagerCore`). I would suggest making it consistent across all.

Reply:
it should return NULL, which will be properly handled in `CexExeGenerator`